backend.go

```diff
@@ -33,5 +34,10 @@ type Session interface {
 	// Add recipient for currently processed message.
 	Rcpt(to string) error
 	// Set currently processed message contents and send it.
-	Data(r io.Reader) error
+	Data(r io.Reader, s LMTPStatusCollector) error
+}
+
+type LMTPStatusCollector interface {
```

Comment:
Question. Why can't we have it as a structure? I can't think of a use case where that interface would be useful.